src/utils/Sorter.js

```diff
@@ -622,13 +622,22 @@ export default Backbone.View.extend({
 
     // Check if the target could accept the source
     let droppable = trgModel.get('droppable');
-    droppable = droppable instanceof Backbone.Collection ? 1 : droppable;
-    droppable = droppable instanceof Array ? droppable.join(', ') : droppable;
-    result.dropInfo = droppable;
-    droppable = isString(droppable) ? this.matches(src, droppable) : droppable;
-    droppable =
-      draggable && this.isTextableActive(srcModel, trgModel) ? 1 : droppable;
-    result.droppable = droppable;
+    if (typeof droppable === 'function') {
+      let res = droppable(src, trg);
```

Comment:
I'd prefer having `droppable(srcModel, trgModel)` as it's easy to get the DOM from the component (`comp.getEl()`) and it's not the same for the opposite sense